.storybook/helpers/files.data.ts

```diff
@@ -59,6 +74,12 @@ export const filesRoot: File[] = [
 
 export function getFileTreeNodeMarkup(files: File[] = filesRoot) {
   return files
-    .map(file => `<clr-tree-node>${file.name}${file.files ? getFileTreeNodeMarkup(file.files) : ''}</clr-tree-node>`)
+    .map(file => {
+      return `
+        <clr-tree-node [clrDisabled]="${file.disabled}" [clrExpanded]="${file.expanded}">
+          ${file.name}
+          ${file.files ? getFileTreeNodeMarkup(file.files) : ''}
+        </clr-tree-node>`;
+    })
```

Comment:
```suggestion
    .map(file => `
      <clr-tree-node [clrDisabled]="${file.disabled}" [clrExpanded]="${file.expanded}">
        ${file.name}
        ${file.files ? getFileTreeNodeMarkup(file.files) : ''}
      </clr-tree-node>
    `)
```

The `return` keyword is not needed.

Reply:
removed in commit 795ffad